staging/src/k8s.io/code-generator/generate-groups.sh

```diff
@@ -73,7 +73,7 @@ fi
 
 if [ "${GENS}" = "all" ] || grep -qw "client" <<<"${GENS}"; then
   echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/clientset"
-  ${GOPATH}/bin/client-gen --clientset-name versioned --input-base "" --input $(codegen::join , "${FQ_APIS[@]}") --output-package ${OUTPUT_PKG}/clientset "$@"
+  ${GOPATH}/bin/client-gen --clientset-name ${CLIENTSET_NAME_VERSIONED:-versioned} --input-base "" --input $(codegen::join , "${FQ_APIS[@]}") --output-package ${OUTPUT_PKG}/clientset "$@"
```

Comment:
Should it be updated as "clientset" in other repos as well: sample-controller, metrics, etc? Is there a reason why some repos have it as "clientset" and some have it as "versioned"?

Reply:
The other way around: this one is off. But because people depend on the old package names, we keep them.

Reply:
:+1: